_data/publications.yml

```diff
@@ -1,3 +1,9 @@
+ - title: "Volumetric Data Fusion of External Depth and Onboard Proximity Data For Occluded Space Reduction"
+   authors: Matthew Strong*, Caleb Escobedo*, Alessandro Roncone
+   year: 2021
+   venue: "4th Workshop on Proximity Perception in Robotics at IROS 2021: Increasing Safety for Human-Robot Interaction using Tactile and Proximity Perception"
```

Comment:
Maybe remove ":Increasing Safety for Human-Robot Interaction using Tactile and Proximity Perception"? It gets too weird otherwise -- see below:

![image](https://user-images.githubusercontent.com/4378663/134211301-26cd11e6-75a9-43c3-b9e7-b4774e21bfed.png)

Too much white space

Reply:
fixed